docs/tutorials/radio.rst

```diff
@@ -0,0 +1,163 @@
+Radio
+-----
+
+Interaction at a distance feels like magic.
+
+Magic might be useful if you're an elf, wizard or unicorn, but such things only
+exist in stories.
+
+However, there's something much better than magic: physics!
+
+Wireless interaction is all about physics: radio waves (a type of
+electromagnetic radiation) have some sort of property (such as their amplitude,
+phase or pulse width) modulated by a transmitter in such a way that information
+can be encoded and, thus, broadcast. When radio waves encounter an electrical
+conductor (i.e. an aerial), they cause an alternating current from which the
+information in the waves can be extracted and transformed back into its
+original form.
+
+Layers upon Layers
+++++++++++++++++++
+
+If you remember, networks are built in layers.
+
+The most fundamental requirement for a network is some sort of connection that
+allows a signal to get from one device to the other. In our networking
+tutorial we used wires connected to the I/O pins. Thanks to the radio module we
+can do away with wires and use the physics summarised above as the invisible
+connection between devices.
+
+The next layer up in the network stack is also different from the example in
+the networking tutorial. With the wired example we used digital on and off to
+send and read a signal from the pins. With the built-in radio on the
+micro:bit the smallest useful part of the signal is a byte.
+
+Bytes
++++++
+
+A byte is a unit of information that (usually) consists of eight bits. A bit is
+the smallest possible unit of information since it can only be in two states:
+on or off.
+
+Bytes work like a sort of abacus: each position in the byte is like a
+column in an abacus - they represent an associated number. In an abacus these
+are usually thousands, hundreds, tens and units (as I would call them in the
```

Comment:
I'd say: "(as one would call them in the UK)" since I don't think you use the first person anywhere else.